### PR TITLE
Sprint3/issue#37 Toolbar

### PR DESCRIPTION
**Pull request by alexisc99**

### CODE REVIEW

File: app/src/main/java/ch/epfl/qedit/model/Question.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
     private final String text;
 
+    private int index;
```
Comment (brunnerant) [inline app/src/main/java/ch/epfl/qedit/model/Question.java]: Do we need this field ? I think we removed it in a previous PR, and now we add it again...

Comment (alexisc99) [reply app/src/main/java/ch/epfl/qedit/model/Question.java]: Actually we don't. I re-added it before seeing the liveData Integer structure you implemented and forgot to remove it, thanks !

**Comment by brunnerant** [comment]
LGTM. Just don't forget to remove the unnecessary commented code before merging.

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```
Comment (nathan96g) [inline app/src/main/res/values/styles.xml]: You cannot really forced all the application to not have the Action Bar, was it the only solution ? Actually what was the problem with adding things to the existing action bar ?

Comment (alexisc99) [reply app/src/main/res/values/styles.xml]: Corrected this ;)

**Pull request merged**